projects/pyyaml/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/yaml/pyyaml"
+language: python
+primary_contact: "david@adalogics.com"
+fuzzing_engines:
+  - libfuzzer
+sanitizers:
+  - address
```

Comment:
Better as:
```
fuzzing_engines:
- libfuzzer
sanitizers:
- address
```

Reply:
I went with the way most projects in oss-fuzz are set up and also how the documentation specifies it https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/#projectyaml

I left it as originally for now due to above reasons - let me know if you would still like the change.